index.html

```diff
@@ -228,36 +221,64 @@ <h2 class="about-head">About me</h2>
           </ul>
         </div>
       </section>
-  <section class="contact">
-  <form action="https://formspree.io/f/xlezlkyw" method="post">
-          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-             <input type="text" placeholder="Full Name" maxlength="30" required>
-             <input type="email" placeholder="Email address" required>
-             <textarea name="msgt1" id="t1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+      <section class="contact">
```

Comment:
- Kindly try to stick as much as possible to the Figma design as requested by previous reviewer, which includes:-
- [ ] The width of the input should be full, use % in style to make it responsive.
- [ ] There is a border line above icons, Please include it.
- [ ] The color of the icons should be dark, please change the color of the icon.
- Your design

![Screenshot from 2022-03-27 02-11-48](https://user-images.githubusercontent.com/26767383/160256699-483146fc-62b5-4ef5-b2d6-784ce1ecd01f.png)

- Figma design
![Screenshot from 2022-03-27 02-11-17](https://user-images.githubusercontent.com/26767383/160256707-2a51192a-733c-49af-8ba7-e95ac9bf7e0d.png)